src/numbers.rs

```diff
@@ -51,21 +51,24 @@ impl IntFloat {
 ///
 /// This is around 2x faster than using `str::parse::<f64>()`
 pub fn float_parse_str(s: &str) -> IntFloat {
-    float_parse_bytes(s.as_bytes())
+    // we could eventually expose the number of decimal digits here, but it's
+    // not needed for now
+    let (float, _decimal_digits_count) = float_parse_bytes(s.as_bytes());
+    float
 }
 
 /// Parse bytes as an float.
-pub fn float_parse_bytes(s: &[u8]) -> IntFloat {
+pub fn float_parse_bytes(s: &[u8]) -> (IntFloat, Option<usize>) {
```

Comment:
please extend the docstrings to explain explain what is being returned.

Reply:
Marking as resolved, as I've reverted this change.